business_register/pep_scoring/rules.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABC, abstractmethod
+import json
+from business_register.models.declaration_models import (Declaration,
+                                                         Property,
+                                                         Vehicle,
+                                                         VehicleRight,
+                                                         Income,
+                                                         Money,
+                                                         PropertyRight,
+                                                         )
+from business_register.models.pep_models import (RelatedPersonsLink, Pep)
+from location_register.models.ratu_models import RatuCity
+
+
+class BaseScoringRule(ABC):
+    def __init__(self, pep):
+        self.pep = pep
+
+    @abstractmethod
+    def calculate_weight(self):
+        pass
+
+
+class IsRealEstateWithoutValue(BaseScoringRule):
+    """
+    Rule 3.1 - PEP03_home
+    weight - 0.4
+    There is no information on the value of the real estate owned by PEP or
+    family members since 2015
+    """
+
+    def calculate_weight(self):
+        family_ids = self.pep.related_persons.filter(
+            to_person_links__category=RelatedPersonsLink.FAMILY,
+        ).values_list('id', flat=True).all()[::1]
```

Comment:
values_list('id', flat=True) already return list
```suggestion
        family_ids = self.pep.related_persons.filter(
            to_person_links__category=RelatedPersonsLink.FAMILY,
        ).values_list('id', flat=True)
```